src/core/gui/PageView.h

```diff
@@ -58,7 +58,7 @@ class XojPageView: public LegacyRedrawable, public PageListener, public xoj::vie
 
 public:
     void addOverlayView(std::unique_ptr<xoj::view::OverlayView>);
-    void rerenderPage() override;
+    void rerenderPage(bool repaint = false) override;
```

Comment:
This default value makes the reviewing process harder: the behaviour is only changed for the occurences of `rerenderPage` that **do not** appear in the diff.
Could you remove the default (or default to true), so that the places where the behaviour changed are easy to find?